Source/WTF/wtf/text/WTFString.h

```diff
@@ -58,14 +58,16 @@ class String final {
 
     // Construct a string with UTF-16 data.
     WTF_EXPORT_PRIVATE String(const UChar* characters, unsigned length);
-    ALWAYS_INLINE String(Span<const UChar> characters) : String(characters.data(), characters.size()) { }
 
     // Construct a string with Latin-1 data.
     WTF_EXPORT_PRIVATE String(const LChar* characters, unsigned length);
     WTF_EXPORT_PRIVATE String(const char* characters, unsigned length);
-    ALWAYS_INLINE String(Span<const LChar> characters) : String(characters.data(), characters.size()) { }
     ALWAYS_INLINE static String fromLatin1(const char* characters) { return String { characters }; }
 
+    // Construct a string from a span object.
+    ALWAYS_INLINE static String fromSpan(Span<const LChar> span) { return String { span.data(), static_cast<unsigned>(span.size()) }; }
+    ALWAYS_INLINE static String fromSpan(Span<const UChar> span) { return String { span.data(), static_cast<unsigned>(span.size()) }; }
```

Comment:
This should be kept grouped in with the other UTF-16 constructors as before.

Reply:
OK.